exonum/src/api/node/private/mod.rs

```diff
@@ -17,16 +17,18 @@
 //! Private API includes requests that are available only to the blockchain
 //! administrators, e.g. view the list of services on the current node.
 
-use std::{collections::HashMap, net::SocketAddr};
+use std::{collections::HashMap, net::SocketAddr, sync::Arc};
 
 use crate::{
-    api::{ApiScope, Error as ApiError},
+    api::backends::actix::{FutureResponse as BackendFutureResponse, RawHandler, RequestHandler},
```

Comment:
I'd add those backend-specific imports inside the `handle_shutdown` function with a comment that we're doing it to support empty request which is not easy in the generic approach, so it will be harder to misuse those features (and as a result get a completely backend-dependent code).